chart/epinio/templates/tekton-buildpacks-task.yaml

```diff
@@ -132,6 +132,7 @@ spec:
         privileged: true
 
     - name: create
+      # TODO: This makes air-gapped installation hard to use. Has to be specified on every push.
```

Comment:
Default builder is defined in code here: https://github.com/epinio/epinio/blob/main/internal/manifest/manifest.go#L18